dev/tools/gen_defaults/lib/checkbox_template.dart

```diff
@@ -31,6 +31,9 @@ class _${blockName}DefaultsM3 extends CheckboxThemeData {
       if (states.contains(MaterialState.selected)) {
         return ${componentColor('md.comp.checkbox.selected.container')};
       }
+      if (states.contains(MaterialState.hovered) || states.contains(MaterialState.focused) || states.contains(MaterialState.pressed)) {
```

Comment:
This should be split per token, and the relevant opacity tokens should be applied as well

Reply:
Split the if condition. We don't have opacity tokens for the outline color here. The relevant opacity tokens in Checkbox are `state-layer.opacity` which are applied in `overlayColor`.